src/screens/ContactScreen/index.js

```diff
@@ -18,6 +18,7 @@ import {Loading} from '../../components';
 import {Search} from './elements';
 import {userPopulate} from '../../service/users';
 import {withInteractionsManaged} from '../../components/WithInteractionManaged';
+import {ANONYMOUS} from '../../hooks/core/constant';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The constant `ANONYMOUS` is imported and used later in the code to compare with the `sourceScreen` parameter. This is a good practice as it avoids hardcoding the string "ANONYMOUS" multiple times throughout the code.
```diff
+ import {ANONYMOUS} from '../../hooks/core/constant';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->